tensorflow/lite/micro/docs/rfc-16x8-quantization-port.md

```diff
@@ -0,0 +1,138 @@
+# TensorFlow Lite for Microcontrollers Port of 16x8 Quantized Operators
```

Comment:
add <!-- mdformat off(b/169948621#comment2) --> to avoid the internal formatting tools from changing things.

https://github.com/tensorflow/tensorflow/blob/a30d20b632b4ffbfd437ccf8ee205fef0917a3eb/tensorflow/lite/micro/CONTRIBUTING.md#L1